### DISCUSSION TITLE
start with sidebar hidden

### DISCUSSION THREAD

**Discussion by AMWarden** [brackets-cont/brackets#147] [Feature Request]
Any chance we could start brackets with the sidebar hidden by default.

**Comment by charlypa** [comment]
@AMWarden Have you meant the right sidebar, which contains tools like `live preview`?

**Reply by AMWarden** [reply]
Hi, I mean the left one...the one with the files...I have the extension to hide/show on mouseover but I have to hide it first on startup in the view menu.

**Reply by charlypa** [reply]
@AMWarden 
![Brackets_ir7ZyXHuAa](https://user-images.githubusercontent.com/1525027/153246700-52fcbf9d-363f-40ad-8eec-62dfa4baa7aa.gif)

**Reply by AMWarden** [reply]
Yes, that's what I mean...but when I restart, I have to do that again.
I was hoping to make that setting sticky on startup.
-Thanks

**Reply by charlypa** [reply]
@AMWarden
![Brackets_Pw5edCd4xx](https://user-images.githubusercontent.com/1525027/153257468-7ba73a85-4a44-4ef1-a89f-79d9c62d9125.gif)
Ideally, it should be set automatically. If it is not set for you, please follow the above image to set it correctly; add the following value
 ` "noDistractions": true`  in `brackets.json` file

**Reply by AMWarden** [reply]
That worked perfectly...thank you so much

**Reply by AMWarden** [reply]
Oh wait...sorry,
But I wanted to keep that bar on the right...the one with the live preview button and such.

**Reply by charlypa** [reply]
Try pressing  `ctrl + alt + p`  together to launch live preview in this mode. Currently, hiding sidebar alone is not supported
